src/styles/blocks/main/card/_rating.scss

```diff
@@ -0,0 +1,30 @@
+.card {
+  &__container--rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  &__rating {
+    font-size: 10px;
+    font-weight: 400;
+  }
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: no-repeat 50% url(../images/star.svg);
+}
+
+@for $index from 1 through $active-stars {
+  .stars--4 > :nth-child(-n + #{$index}) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

Comment:
Reuse all styles from task with stars and move each BEM block to the separate file.